subsys/bluetooth/host/Kconfig

```diff
@@ -350,6 +350,14 @@ config BT_DEVICE_APPEARANCE
 	  consult the following link:
 	  https://www.bluetooth.com/specifications/assigned-numbers
 
+config BT_ID_MAX
+	int "Maximum number of local identities"
+	range 1 36
```

Comment:
Why 36? (I am just curious.)

Reply:
The current settings implementation has a limit of 256 byte values. bt_addr_le_t is 7 bytes and 36 * 7 = 252. That said, I added this before considering IRKs, which are 16 bytes. Also, I failed to consider the fact that there's base64 encoding before writing to flash. So this current maximum is not what you can reach in practice. Most likely it's closer to 10 or so. That said, there's ongoing work to remove the 256 byte value limit from settings. It'd be nice if Kconfig would allow giving only a minimum or maximum but leave the other limit unbounded.